### PR TITLE
add logical-import-prep-stmt

### PR DESCRIPTION
**Pull request by dbsid**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -258,6 +258,9 @@ strategy = ""
 # You can decrease this value to reduce the stress on the cluster due to large transactions.
 # logical-import-batch-rows = 65536
 
+# In Logical Import Mode, this parameter controls whether to use prepared statements and statement cache to improve the performance
```
Comment (qw4990) [inline tidb-lightning/tidb-lightning-configuration.md:261]:
```suggestion
# In Logical Import Mode, this parameter controls whether to use prepared statements and statement cache to improve the performance.
```

**Review by qw4990 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -258,6 +258,9 @@ strategy = ""
 # You can decrease this value to reduce the stress on the cluster due to large transactions.
 # logical-import-batch-rows = 65536
 
+# In Logical Import Mode, this parameter controls whether to use prepared statements and statement cache to improve the performance.
+# logical-import-prep-stmt = false
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:262]:
```suggestion
logical-import-prep-stmt = false
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -258,6 +258,9 @@ strategy = ""
 # You can decrease this value to reduce the stress on the cluster due to large transactions.
 # logical-import-batch-rows = 65536
 
+# In Logical Import Mode, this parameter controls whether to use prepared statements and statement cache to improve performance.
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:261]:
```suggestion
# In Logical Import Mode, this parameter controls whether to use prepared statements and statement cache to improve performance. The default value is `false`.
```

**Review by hfxsd — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**